flaky test failure - tests/server/routes/post-csp

Occasionally, tests/server/routes/post-csp fails like so:

```
node ./node_modules/.bin/intern-client config=tests/intern_server fxaContentRoot=https://accounts.stage.mozaws.n\
et/ fxaProduction=true asyncTimeout=10000
Creating Firefox profile...
{"op":"server.csp","time":"2015-11-23T20:00:00.000Z"}
{"blocked":"http://bing.com","op":"server.csp","time":"2015-11-23T20:00:00.000Z"}
PASS: post-metrics - it drops if no csp-report set (2ms)
{"blocked":"http://bing.com","op":"server.csp","time":"2015-11-23T20:00:00.000Z"}
FAIL: post-metrics - it allows no messages with 0% sample rate  (1ms)
AssertionError: expected true to be false
  at Assertion.<anonymous>  <node_modules/intern/node_modules/chai/chai.js:572:10>
  at Assertion.Object.defineProperty.get  <node_modules/intern/node_modules/chai/chai.js:3515:29>
  at Function.assert.isFalse  <node_modules/intern/node_modules/chai/chai.js:2204:31>
  at Test.suite.it allows no messages with 0% sample rate  [as test]  <tests/server/routes/post-csp.js:59:12>
  at start.then.finally.self.timeElapsed  <node_modules/intern/lib/Test.js:211:24>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/node_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:317:4>
  at Object.run [as _onImmediate]  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at processImmediate [as _immediateCallback]  <timers.js:363:15>
{"blocked":"http://bing.com","op":"server.csp","time":"2015-11-23T20:00:00.000Z"}
{"blocked":"http://bing.com","op":"server.csp","time":"2015-11-23T20:00:00.000Z"}
{"blocked":"http://bing.com","op":"server.csp","time":"2015-11-23T20:00:00.000Z"}
{"blocked":"http://bing.com","op":"server.csp","time":"2015-11-23T20:00:00.000Z"}
{"blocked":"http://bing.com","op":"server.csp","time":"2015-11-23T20:00:00.000Z"}
PASS: post-metrics - it allows all messages with 100% sample rate (1ms)
1/3 tests failed
1/3 tests failed
```

--- a/server/lib/routes/post-csp.js
+++ b/server/lib/routes/post-csp.js
@@ -17,7 +17,7 @@ module.exports = function (options) {
   function isSampledUser() {
     // random between 0 and 100, inclusive
     var rand = Math.floor(Math.random() * (100 + 1));
-    return rand <= options.reportSampleRate;
+    return rand < options.reportSampleRate;
   }
 
   return {

